scripts/glyph_list.json

```diff
@@ -0,0 +1,79 @@
+[
```

Comment:
Does this file need to be manually maintained, or is it being generated by one of the scripts?
If it's manual, it should be added to the SEP V023 draft.

Reply:
It's currently manually maintained, but I'm not sure whether it should eventually be be replaced by the ontology (which would need to be automatically updated, and also expanded to record the version of the spec that last changed each glyph), or whether it should be used when generating the ontology.

Reply:
That's a question for discussion with @goksel that can be deferred until after this release; for now, can you please add a note to SEP V023 that will let us track this in workflow development?

Reply:
I've added a comment.